packages/docusaurus-mdx-loader/src/remark/mdx1Compat/codeCompatPlugin.ts

```diff
@@ -24,10 +24,6 @@ const plugin: Plugin<unknown[], Root> = function plugin(): Transformer<Root> {
 
       node.data.hProperties = node.data.hProperties || {};
       node.data.hProperties.metastring = node.meta;
-
-      // Retrocompatible support for live codeblock metastring
-      // Not really the appropriate place to handle that :s
-      node.data.hProperties.live = node.meta?.split(' ').includes('live');
```

Comment:
Historically, with MDX 1 (Docusaurus v2), the live code block component would receive a "live" prop.

Our users have swizzled the live code block component, so if you remove this line, their component won't receive the `live=true` prop anymore and it might break their site. I'd prefer to keep it, and eventually only remove this for Docusaurus v4.

Reply:
Thanks for the further insight. This area was a bit sensitive for me as I wasn't sure about all the affected parts (also considering tha note above in line 17). 

The normal `CodeBlock` doesn't have a `live` property, it only comes via `docusaurus-theme-live-codeblock`. And I thought there users would rather swizzle the `Playground` component. 

I didn't try all scenarios, but normally swizzling a `CodeBlock` gives you props without `live`. But it might be risky that people might have relied on a hidden live property in older code and declared a `live` property themselves. 

Unfortunately this is the `docusaurus-mdx-loader` package which has no access to the `codeBlockUtils`. So we would need to keep this code as it is. I don't think we can rely on the `metaOptions` easily.